src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/xresilience/StormLcmSpec.groovy

```diff
@@ -82,27 +96,25 @@ class StormLcmSpec extends HealthCheckSpecification {
         flows.each { flow -> flow.validateAndCollectDiscrepancies().isEmpty() }
 
         and: "Database dump"
-        //unstable for parallel runs even when isolated. why?
```

Comment:
It seems like the root cause of flakiness was the 'has' relation for flow_stats .